src/group_operations/addition.jl

```diff
@@ -0,0 +1,206 @@
+
+Compute the differential of the left group multiplication ``λ_g(h) = g$(_math(:∘))h``,
+which simplifies for [`AdditionGroupOperation`](@ref) to ``Dλ_g(h)[X] = X``.
+"""
+
+@doc "$(_doc_diff_left_compose_add)"
+diff_left_compose(G::LieGroup{𝔽,AdditionGroupOperation}, g, h, X) where {𝔽}
+
+@doc "$(_doc_diff_left_compose_add)"
+function diff_left_compose!(G::LieGroup{𝔽,AdditionGroupOperation}, Y, g, h, X) where {𝔽}
+    return copyto!(LieAlgebra(G), Y, X)
+end
+
+_doc_diff_right_compose_add = """
+    diff_right_compose(G::LieGroup{𝔽,AdditionGroupOperation}, h, g, X)
+    diff_right_compose!(G::LieGroup{𝔽,AdditionGroupOperation}, Y, h, g, X)
+
+Compute the differential of the right group multiplication ``ρ_g(h) = h$(_math(:∘))g``,
+which simplifies for [`AdditionGroupOperation`](@ref) to ``Dρ_g(h)[X] = X``.
+"""
+
+@doc "$(_doc_diff_right_compose_add)"
+diff_right_compose(G::LieGroup{𝔽,AdditionGroupOperation}, h, g, X) where {𝔽}
+
+@doc "$(_doc_diff_right_compose_add)"
+function diff_right_compose!(G::LieGroup{𝔽,AdditionGroupOperation}, Y, g, h, X) where {𝔽}
+    return copyto!(LieAlgebra(G), Y, X)
+end
+
+_doc_exp_add = """
+    exp(G::LieGroup{𝔽,AdditionGroupOperation}, e::Identity{AdditionGroupOperation}, X, t=1)
+    exp!(G::LieGroup{𝔽,AdditionGroupOperation}, g, e::Identity{AdditionGroupOperation}, X, t)
+
+Compute the Lie group exponential on a [`LieGroup`](@ref) with an [`AdditionGroupOperation`](@ref).
+This can be computed in-place of `g`.
+
+Since `e` is just the zero-element with respect to the corresponding `+`, the formula reads ``g=0+X=X``.
+"""
+
+@doc "$(_doc_exp_add)"
+Base.exp(
+    ::LieGroup{𝔽,AdditionGroupOperation}, ::Identity{AdditionGroupOperation}, X, t
+) where {𝔽}
+
+@doc "$(_doc_exp_add)"
+function ManifoldsBase.exp!(
+    G::LieGroup{𝔽,AdditionGroupOperation},
+    g,
+    ::Identity{AdditionGroupOperation},
+    X,
+    t::Number=1,
+) where {𝔽}
+    return copyto!(G, g, X)
+end
+
+_doc_identity_element_add = """
+    identity_element(G::LieGroup{𝔽,AdditionGroupOperation})
+    identity_element!(G::LieGroup{𝔽,AdditionGroupOperation}, e)
+
+Return the a point representation of the [`Identity`](@ref),
+which for the [`AdditionGroupOperation`](@ref) is the zero element or array.
+"""
+
+@doc "$(_doc_identity_element_add)"
+identity_element(::LieGroup{𝔽,AdditionGroupOperation}) where {𝔽}
+
+@doc "$(_doc_identity_element_add)"
+function identity_element!(::LieGroup{𝔽,AdditionGroupOperation}, e) where {𝔽}
+    return fill!(e, 0)
+end
+
+_doc_inv_add = """
+    inv(G::LieGroup{𝔽,AdditionGroupOperation}, g)
+    inv!(G::LieGroup{𝔽,AdditionGroupOperation}, h, g)
+
+Compute the inverse group element ``g^{-1}``, which for the [`AdditionGroupOperation`](@ref)
+simplifies to ``-g``. This can be done in-place of `h`.
+"""
+
+@doc "$(_doc_inv_add)"
+inv(G::LieGroup{𝔽,AdditionGroupOperation}, g) where {𝔽}
+
+@doc "$(_doc_inv_add)"
+function inv!(G::LieGroup{𝔽,AdditionGroupOperation}, h, g) where {𝔽}
+    return copyto!(G, h, -g)
```

Comment:
```suggestion
    h .= (-1) .* g
    return h
```

Reply:
I probably said this quite often now, I never know when to use copy and when to use `.=`, so good that you know.